chapters/pre-recorded-stt/features.mdx

```diff
@@ -234,19 +234,36 @@ If you know the context of the audio you're sending, you can provide it in the `
 
 To enhance the precision of transcription, especially for words or phrases that recur often in your audio file, you
  can utilize the `custom_vocabulary` feature in the transcription configuration settings.<br/>
- The custom vocabulary has the following limitation:
-
- - global limit of **10k characters**
- - no more than **100 elements**
- - each element should not contain more than **5 words**
 
 ```json request data
 {
   "audio_url": "YOUR_AUDIO_URL",
-  "custom_vocabulary": ["westeros", "stark", "night's watch"]
+  "custom_vocabulary": true,
+  "custom_vocabulary_config": {
+    "vocabulary": [
+      "Westeros", 
+      { "value": "Stark"}, 
+      { 
+        "value": "Night's Watch",
+        "pronunciations": ["Nightz Vatch"],
+        "intensity": 0.4,
+        "language": "de"
+      }
+    ],
+    "default_intensity": 0.6
+  }
 }
 ```
 
+Use only a string in place of an object is equivalent to use `{ "value": "string" }`
```

Comment:
You can make a snippet with that whole paragraph to avoid duplication.  
See `snippets/` folder for examples.